### PR TITLE
Wagtail 6.0

### PR DESCRIPTION
**Pull request by katdom13**

### CODE REVIEW

File: CHANGELOG.rst [hunk 1]
```diff
@@ -1,9 +1,8 @@
 Unreleased
 ----------
-* Drop support for Wagtail versions 4.2 and 5.0 (@katdom13)
-* Support for Wagtail 5.2 (@katdom13)
```
Comment (alxbridge) [inline CHANGELOG.rst:4]: **question:** Did Wagtail 5.2 support not make it into a previous release of this package? If not, we should still explicitly include this line

File: wagtail_storages/tests/settings.py [hunk 2]
```diff
@@ -46,12 +44,11 @@
...
-if WAGTAIL_VERSION >= (5, 2):
-    # when running tests with Wagtail 5.2+, we need to set the STATIC_URL explicitly,
-    # otherwise the test server will not start.
-    # This happens when running tests against the main branch of Wagtail
-    # so I am assuming that an upcoming release will require this.
-    STATIC_URL = "/static/"
+# when running tests with Wagtail 5.2+, we need to set the STATIC_URL explicitly,
+# otherwise the test server will not start.
+# This happens when running tests against the main branch of Wagtail
+# so I am assuming that an upcoming release will require this.
```
Comment (alxbridge) [inline wagtail_storages/tests/settings.py:50]: **chore:** This comment can be removed now that the configuration's no longer conditional

**Review by alxbridge — COMMENTED** [review]
- Changes reviewed against Wagtail 6.0 upgrade requirements
- Code checked for Django 5.0 compatibility using [django-upgrade](https://github.com/adamchainz/django-upgrade)
- General code review of changes

@katdom13 Just a couple of small documentation-related points: once you've had a chance to look at those you can consider this approved, thanks :+1:

File: CHANGELOG.rst (hunk 1 above)
Comment (katdom13) [reply CHANGELOG.rst]: Hi @alxbridge ,
I retained the previous release notes and put it under a new release. Thanks

File: wagtail_storages/tests/settings.py (hunk 2 above)
Comment (katdom13) [reply wagtail_storages/tests/settings.py]: Removed the comment above

**Pull request merged**